apps/web/lib/api/links/bulk-update-links.ts

```diff
@@ -35,11 +41,17 @@ export async function bulkUpdateLinks(
   const imageUrlNonce = nanoid(7);
 
   const updatedLinks = await Promise.all(
-    linkIds.map((linkId) =>
-      prisma.link.update({
-        where: {
-          id: linkId,
-        },
+    linkIds.map(async (linkId) => {
+      const link = await prisma.link.findUnique({
+        where: { id: linkId },
+      });
+
+      if (!link) {
+        throw new Error(`Link with id ${linkId} not found`);
+      }
```

Comment:
We're validating the link on the route level, so this additional fetch is unnecessary. Can we remove this?

Reply:
I was facing some that's why i used this approach. Let me check again then I'll connect to you

Reply:
Yes, we need to fetch link because bulkUpdateLinksBodySchema contains data as boolean

I'm updateing `?qr=1` in `transform-link.ts`

if still there is any issue regarding to this fetch please tell me

Reply:
@01vishu I just realised we don't support updating `key` and `domain` via `/api/links/bulk`. So there is no need to update the `shortLink` I guess.

Reply:
@devkiran should we fix link search using key and domain only or keep shortLink and use it for search

What is your opinion on this?

Reply:
> I just realised we don't support updating key and domain via /api/links/bulk. So there is no need to update the shortLink I guess.

yup exactly, will remove the changes here